src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -425,6 +425,9 @@ private void doHandle(HttpServletRequest request,
             if (headerName.startsWith("x-amz-meta-")) {
                 continue;
             }
+            if (headerName.startsWith("x-amz-content-")) {
+                continue;
+            }
```

Comment:
Sorry I don't understand; can you show me a test case that this fails with?  Let's use a separate pull request for the test and fix.